bower.json

```diff
@@ -21,18 +21,37 @@
   "homepage": "https://github.com/PolymerElements/gold-zip-input",
   "ignore": [],
   "dependencies": {
-    "iron-validator-behavior": "PolymerElements/iron-validator-behavior#^1.0.0",
-    "iron-form-element-behavior": "PolymerElements/iron-form-element-behavior#^1.0.0",
-    "iron-icon": "PolymerElements/iron-icon#^1.0.0",
-    "paper-input": "PolymerElements/paper-input#^1.0.0",
-    "paper-styles": "PolymerElements/paper-styles#^1.0.0",
-    "polymer": "Polymer/polymer#^1.1.0"
+    "iron-validator-behavior": "PolymerElements/iron-validator-behavior#2.0-preview",
```

Comment:
Please update to new versions and respective resolutions; [reference](https://github.com/PolymerElements/paper-input/blob/master/bower.json)